.github/workflows/test.yml

```diff
@@ -0,0 +1,11 @@
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Image
+        run: docker-compose -f docker-compose.test.yml build
```

Comment:
My opinion: Running Docker inside Docker is a bit weird 😁 seem many people running the test directly on Github Action Runtime. But you could go with this solution 😃 , not sure we could use some nice feature like `Github Action cache` :)

Reply:
@andyduong1920 Sure, I will have a look if I can use github action runtime to use my dockerfile and build image.